src/main/menu/templates/format.js

```diff
@@ -6,7 +6,7 @@ export default function (keybindings) {
     label: 'F&ormat',
     submenu: [{
       id: 'strongMenuItem',
-      label: 'Strong',
+      label: 'Strong Emphasis',
```

Comment:
I think more people would understand what this does if you made the label

```suggestion 
     label: 'Bold',
```

as the term “bold” is very commonly used for this.

Reply:
See above.

Reply:
If you would use "bold" for the strong case, would you also use "emphasis" for the weak case, and not "italic"?

Reply:
Emphasis is currently used by the application menu but I still prefer bold and italic as it's more common and I never seen `Strong Emphasis` as menu label before.